Harmony/Tools/AccessTools.cs

```diff
@@ -1031,7 +1059,7 @@ public static object[] ActualParameters(MethodBase method, object[] inputs)
 			}
 		}
 
-		static FieldRef<T, F> FieldRefAccessInternal<T, F>(FieldInfo fieldInfo, bool needCastclass) where T : class
+		static FieldRef<T, F> FieldRefAccessInternal<T, F>(FieldInfo fieldInfo, bool needCastclass)
```

Comment:
no value check for this change, is this correct?

Reply:
Yes, it's called by all the other FieldRefAccess methods, which already have the checks or hard-code `object` for `T`.